src/seedu/addressbook/data/person/Address.java

```diff
@@ -4,29 +4,43 @@
 
 /**
  * Represents a Person's address in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, some street, #01-01, 123456";
```

Comment:
Is the `EXAMPLE` string used in the code? If not, it should be removed. (Same in `Block`, `PostalCode`, `Street` and `Unit` classes.)